src/css/_variables.scss

```diff
@@ -0,0 +1,19 @@
+
+$font-stack: 'Oswald', sans-serif;
+$primary-background: #1F1F1F;
+$profile-button: 5.6vw;
+$margin-size: .5vw;
+$bright-orange: #F5634B;
+$aqua-blue: #54C6BE;
+$light-purple: #8975B2;
+$pale-yellow: #FBD089;
+
+@mixin button-styles() {
+    background-position: center;
+    background-size: 3.4vw;
+    border: none;
+    display: inline;
```

Comment:
I don't imagine the display: inline will be a problem even though some tags didn't include it, but remember that we put that for them all